elastic/kibana
# [Content management] fix table list flashes table interface when empty

Closes https://github.com/elastic/kibana/issues/159507
Closes https://github.com/elastic/kibana/issues/148557

Flaky test runner https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/2508

### Test instructions
* Install sample web logs data set
* Delete dashboard that ships with sample data set
* switch to another app, like visualize
* Use debugger tools to throttle network connection to "slow 3g"
* Open dashboard application again
* Verify empty table view is never displayed and there is no flickering between views.
